@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/DownloadCitation/index.js

```diff
@@ -1,10 +1,42 @@
 /** @jsx jsx */
 import PropTypes from 'prop-types'
 import { jsx } from 'theme-ui'
+import typy from 'typy'
 
 const DownloadCitation = ({ iiifManifest }) => {
+  let biblio = ''
+  const metadata = {}
+  if (iiifManifest.type === 'Collection') {
+    biblio = iiifManifest.label.en[0] + ', ' + iiifManifest.slug.replace('item/', '') + '. ' +
+      iiifManifest.provider[0].homepage[0].label.en[0] + ', Hesburgh Libraries, University of Notre Dame, South Bend, IN. ' +
+      window.location.href
+  } else {
+    if (typy(iiifManifest, 'provider[0].homepage[0]label.en[0]').isString) {
+      iiifManifest.metadata.map((md) => {
+        metadata[(md.label.en[0].trim())] = md.value.en[0]
+        return metadata
+      })
+      if (iiifManifest.provider[0].homepage[0].label.en[0] === 'Snite Museum of Art') {
+        biblio = metadata.Creator + ', ' + iiifManifest.label.en[0] + ', ' + metadata.Medium + '. ' +
+        'Snite Museum of Art, University of Notre Dame. ' +
+        metadata['Credit Line'] + ', ' + metadata['Accession number'] + '.'
+      }
+      if (iiifManifest.provider[0].homepage[0].label.en[0] === 'Rare Books and Special Collections' | 'University of Notre Dame Archives') {
+        biblio = metadata.Creator + '. ' + iiifManifest.label.en[0] + ', ' + metadata.Date + '. ' +
+        metadata.Collection + '. ' + iiifManifest.provider[0].homepage[0].label.en[0] + ', Hesburgh Libraries, University of Notre Dame, South Bend, IN.' +
+        window.location.href
```

Comment:
`window` can be problematic at build time as it is undefined in Node when it does the initial build of the site. In this particular case it is probably okay to use it because the Download modal is not rendered by default, but if we moved the citation to part of the page that always gets rendered, it will break the build. 
Additionaly we also want to refer the the canonical url (https://marble.library.nd.edu and not http://localhost or https://marble-test.library.nd.edu), so we probably want to use graphql and grab the `siteUrl` and the `slug` from the `iiifManifest`. 

```
`${typy(site, 'siteMetadata.siteUrl').safeString}/${typy(iiifManifest, 'slug').safeString}`
```